html/index.html.in

```diff
@@ -213,6 +214,9 @@
                                                 </button>
                                             </div>
                                             <div class="col-12 col-md-6 form-check d-flex flex-column align-items-end justify-content-end w-auto mt-2" id="translateOptions">
+                                                <button id = "copyText" class="btn btn-secondary mb-2 ">
```

Comment:
Ditto, and remove space before "

Reply:
Forgot to remove the spaces.

Reply:
@TinoDidriksen Sorry, I forgot that. Please check recent commit.

Reply:
@Aayush-hub still need that space before the " gone.

Reply:
@TinoDidriksen Sorry, I can't understand what you requested

Reply:
@Aayush-hub also the space before the other `"`